capabilities.json

```diff
@@ -217,6 +217,37 @@
                         "numeric": true
                     }
                 },
+                "outline": {
```

Comment:
this setting has alreaby been in the code, but it was using only one(default) value. I added opportunity to change this setting